### PR TITLE
Add doctr-versions-menu

### PR DESCRIPTION
**Pull request by hhslepicka**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [ ] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [ ] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

Attn. @goerz would you like to be added as maintainer of this feedstock as well?

### CODE REVIEW

File: recipes/doctr-versions-menu/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  commands:
+    - doctr-versions-menu --help
+
+about:
+  home: https://github.com/goerz/doctr_versions_menu
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'Sphinx extension and command to add a versions menu to Doctr-deployed documentation'
+  doc_url: https://simplejson.readthedocs.io/
```
Comment (synapticarbors) [inline recipes/doctr-versions-menu/meta.yaml:47]: This `doc_url` should be corrected or removed.

File: recipes/doctr-versions-menu/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
+{% set name = "doctr-versions-menu" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/doctr-versions-menu/meta.yaml:9]: `url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/doctr_versions_menu-{{ version }}.tar.gz`

File: recipes/doctr-versions-menu/meta.yaml (hunk 1 above)
Comment (hhslepicka) [reply recipes/doctr-versions-menu/meta.yaml]: Good catch! Addressed via https://github.com/conda-forge/staged-recipes/pull/12250/commits/1f000c20219b59e174dc8bc0e336720c6728c53d

File: recipes/doctr-versions-menu/meta.yaml (hunk 2 above)
Comment (hhslepicka) [reply recipes/doctr-versions-menu/meta.yaml]: Thank you! Addressed via https://github.com/conda-forge/staged-recipes/pull/12250/commits/1f000c20219b59e174dc8bc0e336720c6728c53d

File: recipes/doctr-versions-menu/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
+{% set name = "doctr-versions-menu" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/doctr_versions_menu-{{ version }}.tar.gz
```
Comment (synapticarbors) [inline recipes/doctr-versions-menu/meta.yaml:9]: One more thing, double `url: url:`

Comment (hhslepicka) [reply recipes/doctr-versions-menu/meta.yaml]: Oooops... fixed via https://github.com/conda-forge/staged-recipes/pull/12250/commits/559a5cd430ff236084ebd28ecd9bcc91af26ebce

**Comment by synapticarbors** [comment]
It looks like there are some unicode characters in the cli that aren't being handled properly on Windows:

https://github.com/goerz/doctr_versions_menu/blob/master/src/doctr_versions_menu/cli.py#L250

I'm not 100% sure what to do in this case.

**Comment by hhslepicka** [comment]
I can add a patch to this recipe and remove the unicode. Commit coming soon.

**Comment by goerz** [comment]
> Attn. @goerz would you like to be added as maintainer of this feedstock as well?

Yes, I think so. I haven't created any conda packages in the past, but this looks pretty straightforward. I can't really make any commitments on maintaining the feedstock going forward (if @hhslepicka can do that, that'd be great), but in principle, as the author of the underlying package, it seems like a good idea for me to have maintainer access to the feedstock for future updates

**Comment by hhslepicka** [comment]
@goerz Thank you! I will add you as maintainer at the Feedstock repository https://github.com/conda-forge/doctr-versions-menu-feedstock

**Comment by goerz** [comment]
@synapticarbors Just to follow up on the unicode issue (if you happen to know the answers):

* Is my basic understanding in https://github.com/conda-forge/doctr-versions-menu-feedstock/pull/3#issuecomment-668334298 correct?
* Why did the "staged" recipe run Linux + Mac + Windows, whereas the final feedstock only gets processed on Linux? Is there a way to skip Windows for a "noarch" package in the first place?

**Comment by synapticarbors** [comment]
@goerz -- for the first bullet point, your comment on the linked PR seems correct. For the second, `staged-recipes` always runs all three platforms, but for `noarch: python`, once it gets to the feedstock it only runs on linux, so this is the expected behavior.

**Pull request merged**